web/assets/css/build.less

```diff
@@ -57,14 +57,10 @@
     padding-right: 20px;
     flex-shrink: 0;
     user-select: none;
+    color: @grey-secondary;
+    display: inline-block;
 
-    &::before {
```

Comment:
:thinking: I can't remember why this was done this way before, but it seems to be fine either way. Looks like this makes it possible to test, so :+1: